tf-modules/cumulus/outputs.tf

```diff
@@ -1,9 +1,16 @@
 # API outputs
 
+# TEA-Specific outputs.
+
 output "distribution_bucket_map" {
-  value = module.distribution.distribution_bucket_map
+  value = var.tea_internal_api_endpoint != null ? module.distribution[0].distribution_bucket_map : null
```

Comment:
Let me know what you think of this. Outputs on conditionally created modules aren't the most seamless. A couple things to note:

1. These outputs don't disappear if we're not using TEA. They're just `null`.
2. module.distribution is a list now that it has a `count` param
3. I think the only reason this works at all is because the `distribution` module is now a list with either 1 or 0 members. The Terraform compiler would complain if `module.distribution` doesn't exist at all but it's fine if it's an empty list. It's a little weird.

Reply:
2 and 3 are expected results of making the distribution module conditional. And I think 1 is fine, but need to think about it